docs/ops.html

```diff
@@ -1070,22 +1070,22 @@ <h4><a id="remote_jmx" href="#remote_jmx">Security Considerations for Remote Mon
             Disconnected|SyncConnected|AuthFailed|ConnectedReadOnly|SaslAuthenticated|Expired.</td>
       </tr>
       <tr>
-        <td>Max time to load group metadata</td>
+        <td>Max time to load group metadata (including scheduling time)</td>
```

Comment:
I think the short description is ok. But maybe we could add something like this to the long description:
>  maximum time, in milliseconds, it took to load offsets and group metadata from the consumer offset partitions loaded in the last 30 seconds (including time spent waiting for the loading task to be scheduled)

Reply:
Ack, added this to all the long-form descriptions.